### PR TITLE
Fixed `getting_started.md` setup scripts

### PR DESCRIPTION
**Pull request by tamionv**
Issue #, if available: N/a

Description of changes: Fixes some bugs in the setups scripts in `docs/getting_started.md`.

Signed-off-by: Tamio-Vesa Nakajima <tnk@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -164,26 +164,26 @@ In your **second shell** prompt:
...
     if [ ${arch} = "x86_64" ]; then
       curl --unix-socket /tmp/firecracker.socket -i \
           -X PUT 'http://localhost/boot-source'   \
           -H 'Accept: application/json'           \
           -H 'Content-Type: application/json'     \
-          -d '{
-              "kernel_image_path": \"${kernel_path}\",
-              "boot_args": "console=ttyS0 reboot=k panic=1 pci=off"
-          }'
+          -d "{
```
Comment (andreeaflorescu) [inline docs/getting-started.md]: P.S.: Replacing `'` with `"` is already done in another PR that we are going to merge soon: https://github.com/firecracker-microvm/firecracker/pull/1162

**Comment by aghecenco** [comment]
#1162 was merged, closing this PR
